Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -619,6 +619,30 @@ theorem contMDiffWithinAt_iff_nat {n : ℕ∞} :
   · exact contMDiffWithinAt_infty.2 fun n => h n le_top
   · exact h n le_rfl
 
+theorem contMDiffAt_iff_nat {n : ℕ∞} :
```

Comment:
This chunk could also be a nice short PR (which I'll try hard to review quickly).